files/ODA/ODA_IfcTunnel_Sprint_1_2_Georef_A.ifc

```diff
@@ -0,0 +1,52 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [notYetAssigned]'),'2;1');
+FILE_NAME('e:/ODA_IfcTunnel_Sprint_1_2.ifc','2023-03-31T20:53:49',(''),('Open Design Alliance'),'ODA SDAI 24.4','','810681');
+FILE_SCHEMA(('IFC4X4'));
+ENDSEC;
+
+DATA;
+#1=IFCPERSON('Vov-A6 Lab Tech',$,'Vovchik',$,$,$,$,$);
+#2=IFCORGANIZATION($,'Open Design Alliance',$,$,$);
+#3=IFCPERSONANDORGANIZATION(#1,#2,$);
+#4=IFCORGANIZATION($,'Company 1','File produced in ODA IFC SDK ExIfcTutorials example',$,$);
+#5=IFCAPPLICATION(#4,'0.1 pre alpha','ExIfcTutorials module for ODA IFC SDK','ExIfcTutorials');
+#6=IFCOWNERHISTORY(#3,#5,$,.ADDED.,1680285228,$,$,0);
+#7=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#8=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#9=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#10=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#11=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453292519943295),#10);
+#12=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#13=IFCCONVERSIONBASEDUNIT(#12,.PLANEANGLEUNIT.,'degree',#11);
+#14=IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#15=IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#16=IFCUNITASSIGNMENT((#7,#8,#9,#13,#14,#15));
+#17=IFCPROJECT('0Z0PMrHqb7t9AiL_ogqYgG',#6,'Ifc Tunnel Deployment',$,$,$,$,(#22,#24),#16);
+#18=IFCCARTESIANPOINT((0.,0.));
+#19=IFCCARTESIANPOINT((0.,0.,0.));
+#20=IFCDIRECTION((0.,0.,1.));
+#21=IFCAXIS2PLACEMENT2D(#18,$);
+#22=IFCGEOMETRICREPRESENTATIONCONTEXT('2D','Plan',2,$,#21,$);
+#23=IFCAXIS2PLACEMENT3D(#19,$,$);
+#24=IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,$,#23,$);
+#25=IFCCLASSIFICATION('Classification Source',$,$,'Classification',$,$,$);
+#26=IFCRELASSOCIATESCLASSIFICATION('1r1rECfzj6axXeYEzhiJP6',$,$,$,(#17),#25);
+#27=IFCGROUP('36tmuBeNb27wct1ZhdGxDz',#6,'Just a group',$,$);
+#28=IFCACTORROLE(.USERDEFINED.,'Ifc Tunnel Deployer',$);
+#29=IFCPERSON($,$,'Name of person',$,$,$,(#28),$);
+#30=IFCACTOR('0AAvNrdBrCgv3dTTXg14fX',#6,'An Ifc Tunnel Actor',$,$,#29);
+#31=IFCRELDECLARES('03qpCf5eX2ZQVlwzWNANJZ',#6,'Declarations for IfcProject',$,#17,(#27,#30));
+#32=IFCRELAGGREGATES('3ZWe29_u1Dt8Oiv8$6k16F',$,$,$,#17,(#33));
+#33=IFCSITE('2INsFLAED5Fhi11riLyGtm',#6,'Site',$,$,$,$,$,$,$,$,$,$,$);
+#34=IFCRELAGGREGATES('355RQBrTD97v5RYid2RTJk',$,$,$,#33,(#35));
+#35=IFCTUNNEL('17_Sic8E1EbPfs7i6jQQEA',#6,'Tunnel',$,$,$,$,$,$,.ACCESSTUNNEL.);
+#36=IFCPROJECTEDCRS('EPSG:4326',$,'World Geodetic System (WGS84)',$,$,$,$);
+#37=IFCMAPCONVERSION(#24,#36,10000.,15000.,0.,1.,0.,1.);
+#38=IFCWELLKNOWNTEXT('GEODCRS["WGS 84",  DATUM["World Geodetic System 1984",  ELLIPSOID["WGS 84", 6378137, 298.257223563, LENGTHUNIT["metre", 1]]],  CS[ellipsoidal, 2],  AXIS["Latitude(lat)", north, ORDER[1]],  AXIS["Longitude(lon)", east, ORDER[2]],  ANGLEUNIT["degree", 0.0174532925199433]]"',#36);
```

Comment:
EPSG:4326 is not a projected CRS, but rather a geodetic CRS (see also the WKT string). While such CRS is OK in scenario B, it is not in scenario A and C.